app/redhat_insights/default/data/ui/views/main_dashboard.xml

```diff
@@ -973,13 +977,28 @@
 					"earliest": "-48h",
 					"latest": "now"
 				},
-				"query": "index=redhatinsights AND event_type=any-cve-known-exploit\n  | spath \n  | spath path=events{} output=events\n  | stats by _time, events, application, event_type, account_id, context.display_name\n  | mvexpand events \n  | eval _raw=events\n  | kv\n  | eval known_exploit='payload.known_exploit'\n  | table _time, known_exploit\n  | where known_exploit=True\n  | timechart span=24h aligntime=latest count(known_exploit) as count | appendpipe [| stats count | where count=0 | addinfo | eval time=info_min_time.\" \".info_max_time | makemv time | mvexpand time | table time count | rename time as _time ]"
+				"query": "index=redhatinsights AND event_type=any-cve-known-exploit\n  | spath \n  | spath path=events{} output=events\n  | stats by _time, events, application, event_type, account_id, context.display_name\n  | mvexpand events \n  | eval _raw=events\n  | kv\n  | eval known_exploit='payload.known_exploit'\n  | table _time, known_exploit\n  | where known_exploit=\"true\"\n  | timechart span=24h aligntime=latest count(known_exploit) as count | appendpipe [| stats count | where count=0 | addinfo | eval time=info_min_time.\" \".info_max_time | makemv time | mvexpand time | table time count | rename time as _time ]"
 			},
 			"name": "search_cve_with_know_exploits_24hrs"
 		},
 		"ds_U2K2gqcn_ds_8HzUA9oC_ds_HRRsnn9w": {
 			"type": "ds.search",
 			"options": {
+				"query": "index=redhatinsights AND event_type=any-cve-known-exploit\n  | spath \n  | spath path=events{} output=events\n  | stats by _time, events, application, event_type, account_id, context.display_name\n  | mvexpand events \n  | eval _raw=events\n  | kv\n  | eval known_exploit='payload.known_exploit'\n  | table _time, known_exploit\n  | where known_exploit=\"true\"\n  | timechart span=30d aligntime=latest count(known_exploit) as count | appendpipe [| stats count | where count=0 | addinfo | eval time=info_min_time.\" \".info_max_time | makemv time | mvexpand time | table time count | rename time as _time ]",
+				"queryParameters": {
+					"earliest": "-30d@d",
```

Comment:
The same as above.

Reply:
This line was created by Splunk when I create the new object, and select `Last 30 days` for timestamp.